### PR TITLE
update: add 1.6.20-RC2 EAP details

### PR DESCRIPTION
**Pull request by PetrakovichVictoria**

### CODE REVIEW

File: docs/topics/eap.md [hunk 1]
```diff
@@ -32,6 +32,19 @@ check [our instructions on how to configure your build to support this version](
...
+        <td><strong>1.6.20-RC2</strong>
+            <p>Released: <strong>March 16, 2022</strong></p>
+            <p><a href="https://github.com/JetBrains/kotlin/releases/tag/v1.6.20-RC2" target="_blank">Release on GitHub</a></p>
+        </td>
+        <td>
+            <p>Stabilization of Kotlin 1.6.20-M1 features.</p>
+            <p>For more details, please refer to the <a href="https://github.com/JetBrains/kotlin/releases/tag/v1.6.20-RC2">changelog</a>.</p>
+        </td>
+        <td>
+            <p>Same as for Kotlin 1.6.20-M1, see two rows below.</p>
```
Comment (PetrakovichVictoria) [inline docs/topics/eap.md:45]: The sole line that is different: "see two rows below"

**Review by p7nov — APPROVED** [review]

**Review by koshachy — APPROVED** [review]
Agreed with Pavel

File: docs/topics/eap.md (hunk 1 above)
Comment (PetrakovichVictoria) [reply docs/topics/eap.md:45]: @p7nov sure, thanks!

Comment (PetrakovichVictoria) [inline docs/topics/eap.md:45]:
```suggestion
            <p>Same as for Kotlin 1.6.20-M1, see below.</p>
```

File: docs/topics/eap.md [hunk 2]
```diff
@@ -32,6 +32,19 @@ check [our instructions on how to configure your build to support this version](
         <th>Build highlights</th>
         <th>Recommended kotlinx library versions</th>
     </tr>
+    <tr>
+        <td><strong>1.6.20-RC2</strong>
+            <p>Released: <strong>March 16, 2022</strong></p>
```
Comment (PetrakovichVictoria) [inline docs/topics/eap.md:37]:
```suggestion
            <p>Released: <strong>March 21, 2022</strong></p>
```

File: docs/topics/eap.md [hunk 3]
```diff
@@ -32,6 +32,19 @@ check [our instructions on how to configure your build to support this version](
         <th>Build highlights</th>
         <th>Recommended kotlinx library versions</th>
     </tr>
+    <tr>
+        <td><strong>1.6.20-RC2</strong>
+            <p>Released: <strong>March 21, 2022</strong></p>
```
Comment (PetrakovichVictoria) [inline docs/topics/eap.md:37]:
```suggestion
            <p>Released: <strong>March 22, 2022</strong></p>
```

**Pull request merged**